basic/geometry_math.go

```diff
@@ -6,151 +6,202 @@ import (
 
 	"errors"
 
+	"github.com/go-spatial/geom"
 	"github.com/go-spatial/tegola"
 	"github.com/go-spatial/tegola/maths/webmercator"
 )
 
 // ApplyToPoints applys the given function to each point in the geometry and any sub geometries, return a new transformed geometry.
-func ApplyToPoints(geometry tegola.Geometry, f func(coords ...float64) ([]float64, error)) (G, error) {
+func ApplyToPoints(geometry geom.Geometry, f func(coords ...float64) ([]float64, error)) (geom.Geometry, error) {
 	switch geo := geometry.(type) {
 	default:
-		return G{}, fmt.Errorf("Unknown Geometry: %+v", geometry)
-	case tegola.Point:
+		return nil, fmt.Errorf("Unknown Geometry: %+v", geometry)
```

Comment:
Should this to `%t` so we print the type? Also, let's lowercase the first letter to match the format of the other comments.

Reply:
Using `%T` is a good idea, I'll change that and the casing.